tools/scripts/tidy.toml

```diff
@@ -173,7 +173,7 @@ exit_on_error true
 
 glob_pattern = set "./**/Cargo.toml"
 license = canonicalize LICENSE
-skip_path_prefixes = array "ffi/gn/vendor/" "ffi/gn/third_party_tools/"
+skip_path_prefixes = array "ffi/gn/vendor/" "ffi/gn/third_party_tools/" "ffi/diplomat/c/examples" "provider/testdata/scripts"
```

Comment:
Issue: I think we should err on the side of leaving this list short, even if it means more LICENSE files than we need. We have a number of other crates that aren't intended to be published, and if this crate ever does get published, it should have a LICENSE file. The exclusions currently are only for vendored/third_party crates that are in the gitignore, and I think it should stay that way